qcengine/programs/terachem.py

```diff
@@ -189,13 +189,21 @@ def parse_output(self, outfiles: Dict[str, str], input_model: 'ResultInput') ->
         # TODO Should only return True if TeraChem calculation terminated properly
         output_data['success'] = True
 
+        # return extra files requested by user as extras
+        for extra in input_model.extras.keys():
+            input_model.extras[extra] = outfiles[extra]
+
         return Result(**{**input_model.dict(), **output_data})
 
     def execute(self, inputs, extra_outfiles=None, extra_commands=None, scratch_name=None, timeout=None):
-
+        binaries = []
+        for filename in extra_outfiles:
+            if filename in ['scr/ca0', 'scr/cb0','scr/c0']:
```

Comment:
Should this be `src/*`? We could do something like check if `filename.startswith("src") or filename.startswith("/src") `

Reply:
The scr folder has both binary files like (ca0, cb0, c0, the binary MO coefficients), and many non-binary files like molden, trajectory. I found that it seems like I need to tell the executor which ones are binaries to make sure it can be correctly collected. Otherwise, I got encoding errors.

Reply:
Right, just curious if we needed to make additional files as binary.

Reply:
Yes, I think these are the ones that are generated in all terachem calculations. Some special functionalities may also generate binary files, and I need to figure that our case by case.

Reply:
Ok great! Just checking in.

Looks like your tests are hit by a Pydantic bug that we fixed, but isn't in the current release. I am going to override CI to merge this.

Reply:
Great! Thanks!